bin/update_currencies.php

```diff
@@ -0,0 +1,80 @@
+<?php
+
+use Buzz\Client\ClientInterface;
+use Buzz\Message\Request;
+use Buzz\Message\Response;
+
+require_once __DIR__.'/../vendor/autoload.php';
+
+final class CurrencyUpdater
```

Comment:
Yeah, probably a little but over separated. Since there isn't any control in the flow.